.github/workflows/ci.yml

```
@@ -1,38 +1,39 @@
     branches:
     - main
 
-  pull_request:
-    branches:
-    - main
+  # Run for PRs after docker check
+  workflow_run:
+    workflows: [ "docker-build" ]
+    types:
+      - completed
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

actionlint (1.7.4)

unexpected key "if" for "workflow_run" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)